src/components/Canary/minimal.tsx

```diff
@@ -142,7 +153,7 @@ const MinimalCanaryFC = ({
             {selectedCheck?.canary_id && (
               <HealthCheckEdit check={selectedCheck as HealthCheck} />
             )}
-            {!isCanaryUI && (
+            {!isCanaryUI && !isFeatureDisabled && (
```

Comment:
Can we encapsulate this in a AttachAsEvidenceButton that uses the features context inside (with many different styles for primary, toolbar, menu) ?

Reply:
I have created the button, only one place where moving to the button is trick -> the Topology Dropdown, as the dialog needs to exist outside the dropdown where the button is, otherwise, it doesn't load, since the dialog gets removed from the DOM when the dropdown gets closed.